docs/src/docs/asciidoc/user/clustered-cache.adoc

```diff
@@ -9,77 +9,132 @@ ifdef::notBuildingForSite[]
 include::menu.adoc[]
 endif::notBuildingForSite[]
 
+:imagesdir: ../../../../images/design/clustered
+
 == Introduction
 
 To enable Clustering with Terracotta, firstly you will have to start the http://terracotta.org[Terracotta server] configured with clustered storage.
 And then need a cache manager with clustering capabilities so that the caches can consume the clustered storage.
 
+Detailed instructions on how to configure and start a Terracotta server array can be found elsewhere.
```

Comment:
I would move this to the next section as it relates to TC server config and starting.